packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js

```diff
@@ -214,7 +220,7 @@ export default function InspectedElementWrapper(_: Props) {
           </div>
         </div>
 
-        {canToggleError && (
+        {!hideToggleErrorAction && canToggleError && (
```

Comment:
Can we make this part of the `canToggleError` boolean assignment?

Reply:
Done.